_data/navigation.yml

```diff
@@ -9,6 +9,8 @@ main:
     url: /reference/
   - title: Community
     url: /community/
+  - title: Success Stories
+    url: /success-stories/
```

Comment:
I'm not sure we should create a whole new top-level (and thus top-nav) section. 
@deesprinter , what if we put this under /publications/ but also added a link on the front page, down where there are already some case studies?

Reply:
One of the reasons that led to the success of Kubernetes is having a dedicated page and a top nav menu for case studies/success stories. This is a known formula that people feel better when somebody else is trying out a technology like they are and want to learn from one another. I would strongly vote for a top nav menu item for case studies.

Reply:
I think this is the major topic left along with the related should we move the logos under /community/ to this page. 

I like the Success Stories as a top-nav page because one of the things I think someone wants to see right away if they're evaluating whether or not to use Spinnaker is "what are people doing with it?" 

@fifthposition what're your thoughts?

Reply:
Defer to yourself + @dorbin, as I can see both sides :-)

If we were to go ahead and add a top-level section for "Success Stories", I would be in favor of moving the logos to "Success Stories".

But I also wonder about adding a new top-level for another reason: the top nav is becoming a little lengthy (IMHO). There are currently seven links there. I already wonder about some of the documentation-related ones being a little too fine-grained (e.g. is there a way to put "Concepts" under "Reference" or something?), and "Community" _and_ "Success Stories" might be similar. It'd be nice to have the top-level nav be made of broad and clear divisions, and save smaller distinctions for lower levels of navigation.

Reply:
Dee's right, now that I look at kubernetes.io: Case Studies is top-level. And their success with that is a compelling argument for doing it this way. But Ben's also right: we've got a lot of top-level sections. 

If we weren't otherwise considering re-orging the whole site, I'd say yes, let's consolidate customers with success stories under a top-level section. But because we are getting ready to do a big re-org, the fewer changes we make the better. 

So for now, if we're all ok with it, we can add Dee's success stories at the top level, and defer things like consolidation until we reorganize.

Reply:
Agreed, we'll that we'll take up the bigger organization issues when we meet at Spinnaker Summit.